docs/csharp/whats-new/csharp-8.md

```diff
@@ -241,23 +241,23 @@ int M()
     int x = 7;
     return Add(x, y);
 
-    int Add(int left, int right) => left + right;
+    static int Add(int left, int right) => left + right;
 }
 ```
 
 ## Disposable ref structs
 
-A `struct` declared with the `ref` modifier may not implement any interfaces and so cannot implement <xref:System.IDisposable>. Therefore, to enable a `ref struct` to be disposed, it must have an accessible `void Dispose()` method.
+A `struct` declared with the `ref` modifier may not implement any interfaces and so cannot implement <xref:System.IDisposable>. Therefore, to enable a `ref struct` to be disposed, it must have an accessible `void Dispose()` method. This also applies to `readonly ref struct` declarations.
 
 ## Nullable reference types
 
 Inside a nullable annotation context, any variable of a reference type is considered to be a **nonnullable reference type**. If you want to indicate that a variable may be null, you must append the type name with the `?` to declare the variable as a **nullable reference type**.
 
-For nonnullable reference types, the compiler uses flow analysis to ensure that local variables are initialized to a non-null value when declared. Fields must be initialized during construction. The compiler generates a warning if the variable is not set in all constructors, or an initializer. Furthermore, nonnullable reference types can't be assigned a value that could be null.
+For nonnullable reference types, the compiler uses flow analysis to ensure that local variables are initialized to a non-null value when declared. Fields must be initialized during construction. The compiler generates a warning if the variable is not set a call to any of the available constructors, or an initializer. Furthermore, nonnullable reference types can't be assigned a value that could be null.
```

Comment:
```suggestion
For nonnullable reference types, the compiler uses flow analysis to ensure that local variables are initialized to a non-null value when declared. Fields must be initialized during construction. The compiler generates a warning if the variable is not set by a call to any of the available constructors or by an initializer. Furthermore, nonnullable reference types can't be assigned a value that could be null.
```

Reply:
Thanks for catching that @rpetrusha 

I have clearly reached the point where I've read and modified that sentence too many times in too few days.